src/index.ts

```diff
@@ -434,7 +434,7 @@ export async function cli(args: string[]) {
   const {namedExports, webDependencies, dedupe} = pkgManifest['snowpack'] || {
     namedExports: undefined,
     webDependencies: undefined,
-    dedupe: undefined,
+    dedupe: [],
```

Comment:
Will an empty array work the same as `undefined`?

Reply:
Oh wow, yea it looks like we're overwriting their default of `[]`: https://github.com/rollup/plugins/tree/master/packages/node-resolve#dedupe

Didn't I tell you to replace `[]` with `undefined` in that original PR? Looks like that was a mistake :)